packages/picasso.js/src/core/chart/index.js

```diff
@@ -233,6 +234,22 @@ import componentCollectionFn from './component-collection';
  * @property {number} verticalMoving
  */
 
+/**
+ * A function which returns either (1) false (to specify no progressive rendering used) or an object specifing the data chunk rendered.
```

Comment:
Is this only for the canvas renderer? If so I think that should be stated.

Reply:
Updated!